src/common/components/Toast/constants/ToastText.ts

```diff
@@ -0,0 +1,14 @@
+interface ToastTextType {
+  success: "success";
+  error: "error";
+  warn: "warn";
+  info: "info";
+}
+const ToastText: ToastTextType = {
```

Comment:
이렇게 표현해주면 더 좋을 것 같습니다~
```suggestion
const TOAST_TEXT: ToastTextType = {
```